requirements.txt

```diff
@@ -1,7 +1,7 @@
 torch==1.6.0
 torchvision==0.7.0
 numpy>=1.16.6
-tqdm>=4.43.0
+tqdm>=4.43.0,<4.50.0
```

Comment:
`tqdm` has to be lower than 4.50.0 -- otherwise incompatible with `datasets==1.2.1` (already the latest).

Reply:
I will just try to upgrade it to 4.50.0.